src/frontend/pages/Movies.js

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState } from 'react';
+import { Layout, Input, Row, Col, Card, Tag, Spin, Modal, Typography, Button } from 'antd';
+import { Link } from "react-router-dom";
+import 'antd/dist/antd.css';
+
+const { Content } = Layout;
+const { Search } = Input;
+const { Meta } = Card;
+const TextTitle = Typography.Title;
+
+
+const SearchBox = ({searchHandler}) => {
+    return (
+        <Row>
+            <Col span={12} offset={6}>
+                <Search
+                    placeholder="Search for movies to schedule!"
+                    enterButton="Search"
+                    size="large"
+                    onSearch={value => searchHandler(value)}
+                />
+            </Col>
+        </Row>
+    )
+}
+
+const MovieCard = ({Title, imdbID, Poster, ShowDetails, DetailRequest, ActivateModal}) => {

```

Comment:
We probably want this to be a separate component so we can reuse it on other pages.